helm_deploy/values-dev.yaml

```diff
@@ -19,4 +20,4 @@ generic-service:
 # CloudPlatform AlertManager receiver to route promethues alerts to slack
 # See https://user-guide.cloud-platform.service.justice.gov.uk/documentation/monitoring-an-app/how-to-create-alarms.html#creating-your-own-custom-alerts
 generic-prometheus-alerts:
-  alertSeverity: digital-prison-service-dev
+  alertSeverity: cvl_alerts_non_prod
```

Comment:
need to set up custom alert manager for these, raised an issue [here](https://github.com/ministryofjustice/cloud-platform/issues/4164), will chase on Monday